### PR TITLE
feat: Added the blog on GenAI and RAG

### PR DESCRIPTION
**Pull request by anishbista60**
## Summary

-  This blog is an intro to Gen AI and RAG.
## Basic example

Include a basic example, screenshots, or links.

## Motivation

Why are we doing this? What use cases does it support? What is the expected outcome?

## Checks

- [ ] Read [Create a Pull Request](https://getdoks.org/docs/contributing/how-to-contribute/#create-a-pull-request)
- [ ] Supports all screen sizes (if relevant)
- [ ] Supports both light and dark mode (if relevant)
- [ ] Passes `npm run test`

### CODE REVIEW

**Comment by anishbista60** [comment]
@satrox28 yeah previous one contain too much theory. since, i started learning Gen AI.  That's the reason i written this blog . 

please let me know if improvement is needed.

File: content/en/blog/generative-ai/index.md
Comment (achag033) [inline content/en/blog/generative-ai/index.md:1]: Can we look into possible options to embed this video on the same page?

Comment (satrox28) [reply content/en/blog/generative-ai/index.md:1]: Yes am already checking on it.

I would like to add audio pod cast and video.

Comment (anishbista60) [reply content/en/blog/generative-ai/index.md:1]: > Can we look into possible options to embed this video on the same page?

Do we have this feature at a moment ? or need to develop it so, we can embed video on the same page.

**Comment by satrox28** [comment]
It should be possible on Hugo websites, we don't have to develop but just
have to find right service and integrate,

Thanks & Regards
Sathish Kumar.V

**Comment by anishbista60** [comment]
@achag033 Done.

**Comment by anishbista60** [comment]
Screenshot 2024-11-04 at 9 01 17 AM


In local working fine .

**Comment by achag033** [comment]
Thanks @anishbista60  let me check some content security aspect, its blocking on actual deployment, will fix and update it.

**Comment by anishbista60** [comment]
yeah, it tried with `"<iframe src="https://www.youtube-nocookie.com/embed/NcLAVtQ5H4A" width="675" height="350" allowfullscreen></iframe>"` by direct embedding the `https://www.youtube-nocookie.com/embed/NcLAVtQ5H4A` and still not working let me revert to previous commit. we can adjust (CSP) Settings. Thanks

**Comment by achag033** [comment]
> yeah, it tried with `"<iframe src="https://www.youtube-nocookie.com/embed/NcLAVtQ5H4A" width="675" height="350" allowfullscreen></iframe>"` by direct embedding the `https://www.youtube-nocookie.com/embed/NcLAVtQ5H4A` and still not working let me revert to previous commit. we can adjust (CSP) Settings. Thanks

added one csp update on main, try rebase with main and see if that works on your build.

**Comment by anishbista60** [comment]
not working

**Comment by achag033** [comment]
> not working

ok will check again.

**Comment by anishbista60** [comment]
@achag033 Now, it works fine. 
And i removed the `youtube.com` cause it's no more needed.

**Review by satrox28 — APPROVED** [review]

**Comment by satrox28** [comment]
> > not working
> 
> ok will check again.

Always we have to the add the FQDN matching the target site.

**Comment by anishbista60** [comment]
@satrox28 need to update the date but you merged it.

**Comment by satrox28** [comment]
Oh ok, didn't notice it. I will modify it.

**Comment by anishbista60** [comment]
no worries . I will do it.

**Comment by satrox28** [comment]
Fix: https://github.com/DevOpsCloudJunction/website/pull/95 @anishbista60

**Pull request merged**